modules/appeals_api/spec/fixtures/valid_200995_extra.json

```diff
@@ -31,22 +31,10 @@
             "attributes": {
               "locationAndName": "X-Ray VAMC",
               "evidenceDates": [
-                {
-                  "startDate": "2020-04-10",
-                  "endDate": "2020-04-10"
-                },
-                {
-                  "startDate": "2020-01-02",
-                  "endDate": "2020-02-01"
-                },
-                {
-                  "startDate": "2020-02-20",
-                  "endDate": "2020-02-22"
-                },
-                {
-                  "startDate": "2019-02-02",
-                  "endDate": "2020-02-03"
-                }
+                { "startDate": "2020-04-10", "endDate": "2020-04-10" },
```

Comment:
I condensed these lines while working on this as it helped with readability. I'm happy to revert back, but I think this is much friendlier.